modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -179,6 +186,16 @@ export function toXOnlyPublicKey(b: Buffer): Buffer {
   throw new Error(`invalid key size ${b.length}`);
 }
 
+export function toPlainPublicKey(pubkey: Buffer): Buffer {
```

Comment:
I don't think we should do this as a general practice. If given an x-only key, the even x-y key should only be used in rare cases.

Reply:
Done